### PR TITLE
Adding printDocument: $value method

### PR DESCRIPTION
**Pull request by braedenp-msft**

### CODE REVIEW

**Review by nilakhan — APPROVED** [review]

File: api-reference/beta/api/printdocument-value.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+| Permission type                        | Permissions (from least to most privileged)                  |
+| :------------------------------------- | :----------------------------------------------------------- |
+| Delegated (work or school account)     | PrintJob.Read, PrintJob.Read.All, PrintJob.ReadWrite, PrintJob.ReadWrite.All |
+| Delegated (personal Microsoft account) | Not Supported.                                               |
+| Application                            | PrintJob.Read.All, PrintJob.ReadWrite.All                    |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET https://graph.microsoft.com/beta/print/printers/{id}/jobs/{id}/documents/{id}/$value
```
Comment (Lauragra) [inline api-reference/beta/api/printdocument-value.md:32]: In this section, we don't include the full URL, just the syntax. We show the full URL in the Examples section.

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please also expose the new method in the TOC and the parent resource topic. Thanks!

File: api-reference/beta/resources/printdocument.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Represents a document being printed.
 | Method       | Return Type | Description |
 |:-------------|:------------|:------------|
 | [uploadData](../api/printdocument-uploaddata.md) | None | Upload a single binary segment of the **printDocument**. |
```
Comment (Lauragra) [inline api-reference/beta/resources/printdocument.md:22]: This resource and the uploadData method are not currently exposed in the TOC - please add.

Comment (braedenp-msft) [reply api-reference/beta/resources/printdocument.md:22]: Fixed!

**Comment by braedenp-msft** [comment]
@Lauragra Requested changes made!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**